pkg/utils/crtl_utils.go

```diff
@@ -179,3 +207,41 @@ func CalculateDuration(creationTime time.Time, finishTime time.Time) string {
 	}
 	return finishTime.Sub(creationTime).Round(time.Second).String()
 }
+
+func GenerateRandomRequeueDurationFromEnv() (needReconcile bool, d time.Duration) {
+	d = defaultRuntimeReconcileDuration
+	needReconcile = true
+
+	if RuntimeReconcileDurationEnvVal == "" {
+		return
+	}
+
+	duration, err := strconv.Atoi(RuntimeReconcileDurationEnvVal)
```

Comment:
If this function is called frequently (such as during each Reconcile), repeatedly parsing the string can waste CPU resources. I suggest doing this  once in init function.